### ISSUE TITLE
Erigon 3 stuck in OtterSync while syncing Holesky

### ISSUE THREAD

**Issue by skylenet** [erigontech/erigon#11617]
#### System information

Erigon version: `erigon version 3.00.0-alpha2-6124a58f`

OS & Version: Windows/Linux/OSX: 

Docker image: `ethpandaops/erigon: v3.0.0-alpha2 `

Commit hash: `6124a58f`

Erigon Command (with flags/config):

```
"--datadir=/data",
"--nat=extip:$IP_ADDR",
"--port=30303",
"--http",
"--http.addr=0.0.0.0",
"--http.port=8545",
"--authrpc.jwtsecret=/execution-auth.jwt",
"--authrpc.addr=0.0.0.0",
"--authrpc.port=8551",
"--authrpc.vhosts=*",
"--metrics",
"--metrics.addr=0.0.0.0",
"--metrics.port=6060",
"--http.api=eth,erigon,engine,web3,net,debug,trace,txpool,admin",
"--http.vhosts=*",
"--chain=holesky",
"--prune.mode=full",
"--caplin.discovery.addr=0.0.0.0",
"--caplin.discovery.port=9000",
"--caplin.discovery.tcpport=9000",
"--caplin.checkpoint-sync-url=https://checkpoint-sync.holesky.ethpandaops.io/eth/v2/debug/beacon/states/finalized",
"--beacon.api=beacon,config,debug,events,node,validator,lighthouse",
"--beacon.api.addr=0.0.0.0",
"--beacon.api.port=5052",
"--beacon.api.cors.allow-origins=*"
```

Consensus Layer:

Built-in Caplin

Consensus Layer Command (with flags/config):

Chain/Network: 

Holesky

#### Expected behaviour

Node to sync up to the tip of the chain

#### Actual behaviour

Stuck in OtterSync. Full logs are attached at the bottom. 

```
[INFO] [08-14|06:20:27.850] [1/6 OtterSync] downloading header-chain progress="99.90% 1.2GB/1.2GB" time-left=999hrs:99m total-time=16h12m0s download=0B/s flush=0B/s hash=0B/s complete=0B/s upload=0B/s peers=0 files=58 metadata=40/58 connections=0 alloc=6.8GB sys=13.2GB
```

#### Backtrace

- [full logs](https://github.com/user-attachments/files/16613215/erigon-logs-holesky-stuck.txt)

**Comment by AskAlexSharov** [comment]
Fixed by https://github.com/erigontech/erigon/pull/11532

**Comment by skylenet** [comment]
> Fixed by #11532

Thanks! Yes, I just tried the most recent main branch (`a264b9f`) and it seems to work:

```
[INFO] [08-14|19:07:39.321] [1/6 OtterSync] DONE                     in=19m15.038116065s block=2122999
```
